CMakeLists.txt

```diff
@@ -13,21 +13,40 @@ if(NOT COMMAND add_llvm_library)
         # We need to match the build environment for LLVM:
         # In particular, we need C++11 and the -fno-rtti flag
         set(CMAKE_CXX_STANDARD 14)
-	if(CMAKE_BUILD_TYPE MATCHES "Debug")
-		set(CMAKE_CXX_FLAGS "-fPIC -std=gnu++14 -g -O0 -fno-rtti -Wno-deprecated -DHAVE_IEEE_754 -DSIZEOF_VOID_P=8 -DSIZEOF_LONG=8")
-	else()
-		set(CMAKE_CXX_FLAGS "-fPIC -std=gnu++14 -O3 -fno-rtti -Wno-deprecated -DHAVE_IEEE_754 -DSIZEOF_VOID_P=8 -DSIZEOF_LONG=8")
-	endif()
-	set(CMAKE_C_FLAGS "-fPIC")
-
-	find_package(LLVM REQUIRED CONFIG)
+        set(CMAKE_CXX_STANDARD_REQUIRED ON)
+        # add -std=gnu++14
+        set(CMAKE_CXX_EXTENSIONS ON)
+
+        add_compile_definitions(HAVE_IEEE_754)
```

Comment:
Thanks for your contribution:) Could this be moved to the CMakeList under CUDD since these marcos are only used there?

Reply:
Updated, please have a look.